### ISSUE TITLE
Splitting RTMP Service Methods into separate pages for each service?

### ISSUE THREAD

**Issue by frostycpu** [loldevs/leaguespec#13]
Would it be a good idea to split https://github.com/loldevs/leaguespec/wiki/RTMP-Service-Methods into multiple pages?

**Comment by Divi** [comment]
I don't think so, the page is not too long. And the best advantage is that someone can copy/paste the whole page in a few second, cloning the repository is not needed.

**Comment by jaagupkymmel** [comment]
I think the best way would be to split it into multiple pages, but retain one parent page, which would include the basic description of its methods (the tables) as well as links to each services' page.

**Comment by Divi** [comment]
Tables may be desynchronized (the table must be duplicated) if someone forget to update the main page, or the dedicated page.

**Comment by frostycpu** [comment]
I think desynchronization shouldn't be that big of an issue. I'd really like to add some more information to the page but it's tedious right now to add anything to that big page. jaagupkymmel's suggestion is pretty good in my opninion.

**Comment by Divi** [comment]
I didn't think about new data, :+1: then :)

**Comment by Valandur** [comment]
+1 for this, although some pages might have barely any content...

**Comment by Valandur** [comment]
I moved the first five services into their own pages. Waiting for feedback now, if it's good I guess we can move the rest aswell.
We'll still have to think about what we want on the parent page, and what not.

**Comment by jaagupkymmel** [comment]
Looks good!

**Comment by frostycpu** [comment]
Ya looks good so far =]
